remote-api/src/main/java/org/alfresco/rest/api/model/Tag.java

```diff
@@ -130,6 +130,20 @@ public boolean equals(Object obj)
 				return false;
 		} else if (!nodeRef.equals(other.nodeRef))
 			return false;
+		if(tag==null)
+		{
+			if(other.tag != null)
+				return false;
+		}
+		else if (!tag.equals(other.tag))
+			return false;
+		if (count == null)
+		{
+			if (other.count != null)
+				return false;
+		}
+		else if (!count.equals(other.count))
+			return false;

```

Comment:
Update to `equals` without corresponding update to `hashCode` but I think it's fine in this case.